injitest/src/main/resources/config/Kernel.properties

```diff
@@ -24,32 +24,32 @@ policytest_userName=111998
 #IDREPO
 mosip_idrepo_app_id=idrepo
 mosip_idrepo_client_id=mosip-idrepo-client
-mosip_idrepo_client_secret=HafqST5Aa6mm05lH
+mosip_idrepo_client_secret=AeM2TwFuuZygCuvO
```

Comment:
Remove all the secrets and password. Access these through env variable in pipeline 
cc @lsivanand